docs/src/pages/system/styled/styled.md

```diff
@@ -93,3 +93,94 @@ const styled = createStyled({ defaultTheme });
 
 export default styled;
 ```
+
+## Difference with the `sx` prop
+
+The `styled` function is an extension of the `styled` utility provided by the underlying styled engine used, emotion or styled-components.
+It is guaranteed that it will produce the same output as the `styled` function coming from the styled engine for the same input.
+
+The [`sx`](/system/the-sx-prop/) prop, on the other hand, is a new way of styling your components, focused on fast customization. `styled` is a function, while `sx` is a prop of the MUI components.
+
+Therefore, you will notice the following differences:
+
+### `sx` provides more shortcuts than `styled`
+With `styled`:
+```js
+const MyStyledButton = styled("button")({
+  mx: 1, // ❌ don't use this! This shortcut is only provided by the `sx` prop
+})
+```
+With `sx`:
+```js
+const MyStyledButton = (props) => (
+  <button sx={{
+    mx: 1, // ✔️ this shortcut is specific to the `sx` prop,
+  }}>
+     {props.children}
+  </button>
+})
+```
+### The style definition slightly varies
+With `styled`:
+```js
+const MyStyledButton = styled("button")({
+  padding: 1, // means "1px", NOT "theme.spacing(1)"
+})
+```
+
+With `sx`:
+```js
+const MyStyledButton = (props) => (
+  <button sx={{
+    padding: 1 // means "theme.spacing(1)", NOT "1px"
+  }}>
+     {props.children}
+  </button>
+})
+```
+
+### Patterns of how to use props are different
+
+With `styled`:
+```js
+const MyStyledButton = styled("button")((props) => ({
+  backgroundColor: props.myBackgroundColor
+}))
+
+```
+
+With `sx`:
+```js
+const MyStyledButton = (props) => (
+  <button sx={{backgroundColor: props.myCustomColor}}>
+     {props.children}
+  </button>
+})
+```
+
+### Parameter of each field is different, when using function
```

Comment:
```suggestion
### Parameter when using function are different for each field
```

Reply:
Yes, sorry we missed this - feel free to submit a quick fix. Editing the file in GitHub and submitting a PR from there might be simplest.